src/org/zaproxy/zap/GuiBootstrap.java

```diff
@@ -345,15 +340,37 @@ private static void setDefaultViewLocale(Locale locale) {
      * {@link java.net.InetAddress InetAddress}) preventing some ZAP options from being correctly applied.
      */
     private void setupLookAndFeel() {
-        if (lookAndFeelSet) {
-            return;
+    	
+    	String lookAndFeelClassnameFromCommandline = System.getProperty("swing.defaultlaf");
+    	OptionsParam options = Model.getSingleton().getOptionsParam();
```

Comment:
This should be done later, only if the L&F was not set through the command line.